Projects/Contracts/Sources/View/TerminationSuccessScreen.swift

```diff
@@ -29,9 +29,30 @@ public struct TerminationSuccessScreen: View {
         .padding(.bottom, -100)
 
         hButton.LargeButtonFilled {
+
+            let currentMarket = Localization.Locale.currentLocale.market
+            var surveyURL = ""
```

Comment:
You could do something like this instead of mutating the var I think:

```suggestion
            var surveyURL: URL {
              switch currentMarket { ...cases }
            }
```

😄

Reply:
What about?

   `            var surveyURL: URL? {
                switch currentMarket {
                case .se:
                    return URL(string: "https://hedvigapp.typeform.com/to/YHVVx1#memberid=xxxxx&contract_id=xxxxx&contract_name=xxxxx&terminated_contracts_count=xxxxx")
                case .no:
                    return URL(string: "https://hedvigapp.typeform.com/to/a7aRZzir#memberid=xxxxx")
                case .dk:
                    return URL(string: "https://hedvigapp.typeform.com/to/dk9Dj7S8#memberid=xxxxx")
                case .fr:
                    return nil
                }
            }
            
            if let surveyURL {
                UIApplication.shared.open(surveyURL)
            }`

Reply:
var surveyURL: URL? {
                switch currentMarket {
                case .se:
                    return URL(string: "https://hedvigapp.typeform.com/to/YHVVx1#memberid=xxxxx&contract_id=xxxxx&contract_name=xxxxx&terminated_contracts_count=xxxxx")
                case .no:
                    return URL(string: "https://hedvigapp.typeform.com/to/a7aRZzir#memberid=xxxxx")
                case .dk:
                    return URL(string: "https://hedvigapp.typeform.com/to/dk9Dj7S8#memberid=xxxxx")
                case .fr:
                    return nil
                }
            }
            
            if let surveyURL {
                UIApplication.shared.open(surveyURL)
            }`
`

Reply:
👌